fontforge/woff.c

```diff
@@ -233,11 +233,7 @@ SplineFont *_SFReadWOFF(FILE *woff,int flags,enum openflags openflags, char *fil
     rewind(woff);
     {
         int signature = getlong(woff);
-        if ( signature==CHR('w','O','F','2') ) {
```

Comment:
Was this being called on woff2 files?

Reply:
No, this is dead code; `ReadSplineFont` already checks the signature, so the only way you could have reached this function is if the signature was `wOFF`.